javascript/javascript3/week1/gif.js

```diff
@@ -0,0 +1,13 @@
+const searchGif = () => {
+    const searchWord = document.getElementById("search-input").value;
```

Comment:
What if searchWord.value is "" ? Then you have to let user know that input is empty and he has to type there name 🙂